doc/appendix-de-kassensichv/receipt-case-definitions/receipt-case-definitions.md

```diff
@@ -0,0 +1,22 @@
+## Receipt Case Definitions
+
+This chapter expands on the definitions of Receipt Cases covered in Chapter ["Receipt Case Definitions"](../../general/receipt-case-definitions/receipt-case-definitions.md) of the General Part, with country specific information applicable to the German market.
+
+### Voucher (Service/Product)
+
+The issuance and redemption of vouchers are processes with DSFinV-K requirements. There are two kinds of vouchers to be considered:
+
+- Single purpose vouchers (DE: Einzweckgutscheine)
+- Multi purpose vouchers (DE: Mehrzweckgutscheine)
+
+The difference is that for single purpose vouchers the VAT rate is known already at issuance because it refers to a single purpose (service or product). On the other hand, in case of multi purpose vouchers the VAT rate is not known because it can be redeemed for different purposes (services and/or products).
+
+Handling the issuance and redemption of single purpose vouchers depends on the type of business that the pos operator runs:
+
+1. If she runs a business with profit determination in regards of **§ 4 Abs. 1 EStG** than she must not consider the VAT rate at issuance of the voucher and the charge item case (`ftChargeItemCase`) `0x4445000000000065` should be used at issuance. At redemption a charge item case considering the VAT rate must be used (one of this: `0x4445000000000069`, `0x444500000000006A`, `0x444500000000006B`, `0x444500000000006C`, `0x444500000000006E` or `0x444500000000006F`). 
```

Comment:
1. The classification of §4 1 EStG (or § 4 3) has no direct relation to the VATaxation. 
I propose to delete this examples and link to or translate page 50 of https://fiskaltrust.de/wp-content/uploads/sites/5/2020/04/20200302_DSFinV_K-2.1.pdf

2. This EU-rule is not a speciality of DE

3. Other comments:
a) Why "she"?
b) on 2nd occurance change "charge item case" to "ftChargeItemCase" or at every occurence: "charge item case (´ftChargeItemCase´)"? )

Reply:
1. as discussed in our call (AKO, CRO): we remove details about §4 1 EStG (or § 4 3) in regards of taxation- because it can not be described in very detail- but we add an example on how to use it in terms of charge and pay items. 

2. as discussed in our call (AKO, CRO): we first add it her for Germany and then optimise it to be part of the general description for all EU countries.

3. 
a) (AKO:, CRO): ok, gender
b) (AKO, CRO): will be fixed to ftChargeItemCase

Reply:
Thank you, resolved here: https://github.com/christain-rogobete/interface-doc/commit/f83fb81bad8268214df16ce4622c73df42d4094c